lib/main.dart

```diff
@@ -0,0 +1,7 @@
+import 'package:qr_code/bootstrap.dart';
+import 'package:qr_generator_api/qr_generator_api.dart';
+
+Future<void> main() async {
+  final qrCodeApi = QrGeneratorApi();
+  boostrap(qrCodeApi: qrCodeApi);
```

Comment:
**question**: why did you use the `bootstrap` function rather than defining the intialization code in the `main` entrypoint? which decision did you make? 🙂

Reply:
There isn't a special reason. I just wanted to keep the main function more readable as possible :)